assets/src/edit-story/components/library/panes/media/local/karma/mediaFetching.karma.js

```diff
@@ -49,7 +49,7 @@ describe('MediaPane fetching', () => {
   async function expectMediaElements(expectedCount) {
     let mediaElements;
     await waitFor(() => {
-      mediaElements = localPane.querySelectorAll('[data-testid=mediaElement]');
+      mediaElements = localPane.querySelectorAll('.mediaElement');
```

Comment:
Could also use `[data-testid^="mediaElement"]` here

Reply:
No we can't. MediaElement was changed to MediaElement-type. By your request.

Reply:
Fixed.